### ISSUE TITLE
Search results titles should be heading level 2 for easy access.

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse#793]
This is already happening for boxed audio results, but image results should have their `sr-only` titles updated to use `h2` instead of `span`.

**Comment by sarayourfriend** [comment]
@obulat You :wave: reacted in Make slack when I asked about this issue. Are you planning on taking it on and if so can you assign yourself to it? If not I'll keep looking for another volunteer :slightly_smiling_face:
